src/runtime/js/ts.ui/bars/bars-api@tradeshift.com/models/footer/ts.ui.FooterBarModel.js

```diff
@@ -154,16 +154,23 @@ ts.ui.FooterBarModel = (function using(PagerModel, ActionModel, chained) {
 		 * TODO: Move this thing to the general {ts.ui.ToolBarModel}
 		 * @returns {this|ts.ui.ActionModel}
 		 */
-		collabbutton: chained(function(onclick) {
+		collabbutton: chained(function(args) {
 			var actions = this.bufferbar.actions;
 			if (arguments.length) {
 				actions.clear();
-				if (onclick !== null) {
+				if (args !== null) {
 					actions.push({
 						label: ts.ui.Footer.localize().collaboration,
 						icon: 'ts-icon-collaboration',
-						onclick: onclick
+						onclick: args.onclick
 					});
+					if (args.notification === true) {
```

Comment:
```
this.centerbar.collabbutton = !!args.notification;
this.backupbar.collabbutton = !!args.notification;
```
Could be easier.

Reply:
Could be, but if I enter `'asdf'` as a value it'll show the notification state, which I don't think we want.